client/core/notification.go

```diff
@@ -14,20 +14,21 @@ import (
 
 // Notifications should use the following note type strings.
 const (
-	NoteTypeFeePayment   = "feepayment"
-	NoteTypeWithdraw     = "withdraw"
-	NoteTypeOrder        = "order"
-	NoteTypeMatch        = "match"
-	NoteTypeEpoch        = "epoch"
-	NoteTypeConnEvent    = "conn"
-	NoteTypeBalance      = "balance"
-	NoteTypeSpots        = "spots"
-	NoteTypeWalletConfig = "walletconfig"
-	NoteTypeWalletState  = "walletstate"
-	NoteTypeServerNotify = "notify"
-	NoteTypeSecurity     = "security"
-	NoteTypeUpgrade      = "upgrade"
-	NoteTypeDEXAuth      = "dex_auth"
+	NoteTypeFeePayment       = "feepayment"
+	NoteTypeWithdraw         = "withdraw"
+	NoteTypeOrder            = "order"
+	NoteTypeMatch            = "match"
+	NoteTypeEpoch            = "epoch"
+	NoteTypeConnEvent        = "conn"
+	NoteTypeBalance          = "balance"
+	NoteTypeAssetRatesUpdate = "assetRatesUpdate"
```

Comment:
Make the name shorter, please. Maybe `NoteTypeFiatRates` or something. And let's keep with the trend here and avoid camel-case for the value. Maybe `"fiatrateupdate"`.